pkg/controllers/state/cluster.go

```diff
@@ -67,11 +79,13 @@ type Node struct {
 	// of the Node to identify the remaining resources that we expect future daemonsets to consume.  This is already
 	// included in the calculation for Available.
 	DaemonSetRequested v1.ResourceList
-
 	// HostPort usage of all pods that are bound to the node
 	HostPortUsage *HostPortUsage
+	// Provisioner is the provisioner used to create the node.
+	Provisioner *v1alpha5.Provisioner
 
-	podRequests map[types.NamespacedName]v1.ResourceList
+	podRequests  map[types.NamespacedName]v1.ResourceList
+	InstanceType cloudprovider.InstanceType
```

Comment:
How expensive would it be to retrieve these (cached) per node. This will avoid the conflict w/ @dewjam

Reply:
We cache them for 5 minutes, so 12 times per hour * # of provisioners should be the max number of requests. Shouldn't be terrible.